unison-src/transcripts/project-outputs/.github/ISSUE_TEMPLATE/bug_report.output.md

```diff
@@ -1,12 +1,10 @@
------
-
-name: Bug report
+---
 about: Create a report to help us improve
-title: ''
-labels: bug
 assignees: ''
-
------
+labels: bug
+name: Bug report
+title: ''
+---
```

Comment:
Hah, did it reorder and alphabetize these? I guess that's fine.

Reply:
Ah, I missed that. I don’t know if `yaml` supports preserving ordering, but I do know that YAML doesn’t allow ordering to be significant. So, while it would be nice to preserve what people wrote, it isn’t allowed to affect the behavior.